### ISSUE TITLE
Docker repository does not exist 

### ISSUE THREAD

**Issue by kaykurokawa** [coinbase/toshi#165]
Dockerfile specifies obtaining docker image from 

quay.io/coinbase/rails:latest 

But this repository does not seem to exist anymore

**Comment by DoctorBit** [comment]
I gave up on both vagrant and docker.  Toshi works fine if you just install the Postgresql server directly.  Docker doen't really save any space anyway.  I think they are doing misleading advertising because they make it seem as if docker applications share the os code between them, but that only works if the apps are using the exact same os version.  In every practical use I've seen of docker, each app uses its own os version, in which case you might as well use virtual machines.  Also, how do you do daily os system updates for security?  You have to ship and install an entire base os image with each application every day! Docker doesn't make any sense to me.

**Comment by nicholalexander** [comment]
yes - it appears the dockerfile does not work because of this reference.  only solution I have found is to use this:
https://github.com/coinbase/toshi/blob/1f77ecc73d05984496ef487881880e1e70f7d624/Dockerfile

**Comment by hudon** [comment]
Latest docker-related merges should fix these issues. Please try upgrading to the latest Docker and Docker Compose then:
```
$ docker-compose build
$ docker-compose start db
$ docker-compose run web bundle exec rake db:create
$ docker-compose up
$ open web:5000
```

**Comment by chevdor** [comment]
I have used the instructions provided by @hudon above and it was very smooth with docker.
I just got an error after running ```docker-compose run web bundle exec rake db:create``` but a second run somehow fixed it.
